appservice/src/deploy/wizard/deployZip/WaitForDeploymentToCompleteStep.ts

```diff
@@ -3,13 +3,48 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { AzureWizardExecuteStep, nonNullProp } from "@microsoft/vscode-azext-utils";
-import { Progress } from "vscode";
+import { ActivityChildItem, ActivityChildType, activityFailContext, activityFailIcon, activityProgressContext, activityProgressIcon, activitySuccessContext, activitySuccessIcon, AzureWizardExecuteStep, createContextValue, ExecuteActivityOutput, nonNullProp } from "@microsoft/vscode-azext-utils";
+import { l10n, Progress } from "vscode";
 import { InnerDeployContext } from "../../IDeployContext";
 import { waitForDeploymentToComplete } from "../../waitForDeploymentToComplete";
 
 export class WaitForDeploymentToCompleteStep extends AzureWizardExecuteStep<InnerDeployContext> {
+    public stepName: string = 'WaitForDeploymentToCompleteStep';
     public priority: number = 210;
+    public createSuccessOutput(context: InnerDeployContext): ExecuteActivityOutput {
+        return {
+            item: new ActivityChildItem({
+                contextValue: createContextValue([activitySuccessContext, context.site.id]),
+                label: l10n.t('Finished deployment pipeline.'),
```

Comment:
Can we make the tree item labels for these children present tense

Reply:
It's a bit odd if it's present tense in this case. Deployment is kind of roughly broken into two steps-- deploying the package itself and then pinging the logs to see when it completed. 

I know generally we adopt a pattern where the tree label doesn't change throughout progress/success, but I felt like in this case, it doesn't make a lot of sense to do that.